src/test/shell/bazel/bazel_hermetic_sandboxing_test.sh

```diff
@@ -177,6 +343,51 @@ function test_writing_input_file() {
   expect_log "original text input"
 }
 
+# Test that invalid write of input file is detected, when file is accessed via resolved symlink.
+function test_overwrite_via_resolved_symlink() {
+  [ "$PLATFORM" != "darwin" ] || return 0
```

Comment:
Instead of having this check in every test, could you make the whole test not be run on darwin?